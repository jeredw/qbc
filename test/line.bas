line (0, 0)-(10, 10)
line step(0, 0)-step(10, 10)
line -(10, 10),
line (0, 0)-(10, 10), 15, B, &hff00
line (0, 0)-(10, 10), 15, B
line (0, 0)-(10, 10), 15,  , &hff00
line (0, 0)-(10, 10), 15
line (0, 0)-(10, 10), 15,
line (0, 0)-(10, 10), , B, &hff00
line (0, 0)-(10, 10), , B
line (0, 0)-(10, 10), , , &hff00
line (0, 0)-(10, 10), ,
