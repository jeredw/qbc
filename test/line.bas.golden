(program 
  (statement 
    (line_statement line 
      ( 
        (expr 
          (literal 0)) , 
        (expr 
          (literal 0)) ) - 
      ( 
        (expr 
          (literal 10)) , 
        (expr 
          (literal 10)) ))) \n 
  (statement 
    (line_statement line step 
      ( 
        (expr 
          (literal 0)) , 
        (expr 
          (literal 0)) ) - step 
      ( 
        (expr 
          (literal 10)) , 
        (expr 
          (literal 10)) ))) \n 
  (statement 
    (line_statement line - 
      ( 
        (expr 
          (literal 10)) , 
        (expr 
          (literal 10)) ) ,)) \n 
  (statement 
    (line_statement line 
      ( 
        (expr 
          (literal 0)) , 
        (expr 
          (literal 0)) ) - 
      ( 
        (expr 
          (literal 10)) , 
        (expr 
          (literal 10)) ) , 
      (expr 
        (literal 15)) , 
      (box_style B) , 
      (expr 
        (literal &hff00)))) \n 
  (statement 
    (line_statement line 
      ( 
        (expr 
          (literal 0)) , 
        (expr 
          (literal 0)) ) - 
      ( 
        (expr 
          (literal 10)) , 
        (expr 
          (literal 10)) ) , 
      (expr 
        (literal 15)) , 
      (box_style B))) \n 
  (statement 
    (line_statement line 
      ( 
        (expr 
          (literal 0)) , 
        (expr 
          (literal 0)) ) - 
      ( 
        (expr 
          (literal 10)) , 
        (expr 
          (literal 10)) ) , 
      (expr 
        (literal 15)) , , 
      (expr 
        (literal &hff00)))) \n 
  (statement 
    (line_statement line 
      ( 
        (expr 
          (literal 0)) , 
        (expr 
          (literal 0)) ) - 
      ( 
        (expr 
          (literal 10)) , 
        (expr 
          (literal 10)) ) , 
      (expr 
        (literal 15)))) \n 
  (statement 
    (line_statement line 
      ( 
        (expr 
          (literal 0)) , 
        (expr 
          (literal 0)) ) - 
      ( 
        (expr 
          (literal 10)) , 
        (expr 
          (literal 10)) ) , 
      (expr 
        (literal 15)) ,)) \n 
  (statement 
    (line_statement line 
      ( 
        (expr 
          (literal 0)) , 
        (expr 
          (literal 0)) ) - 
      ( 
        (expr 
          (literal 10)) , 
        (expr 
          (literal 10)) ) , , 
      (box_style B) , 
      (expr 
        (literal &hff00)))) \n 
  (statement 
    (line_statement line 
      ( 
        (expr 
          (literal 0)) , 
        (expr 
          (literal 0)) ) - 
      ( 
        (expr 
          (literal 10)) , 
        (expr 
          (literal 10)) ) , , 
      (box_style B))) \n 
  (statement 
    (line_statement line 
      ( 
        (expr 
          (literal 0)) , 
        (expr 
          (literal 0)) ) - 
      ( 
        (expr 
          (literal 10)) , 
        (expr 
          (literal 10)) ) , , , 
      (expr 
        (literal &hff00)))) \n 
  (statement 
    (line_statement line 
      ( 
        (expr 
          (literal 0)) , 
        (expr 
          (literal 0)) ) - 
      ( 
        (expr 
          (literal 10)) , 
        (expr 
          (literal 10)) ) , ,)) \n <EOF>)
